After launching the task, the Stopped:AStatus137 message is repeated and the deployment does not complete

The ECS DiftAPI container does not start properly. To be precise, it starts once, but immediately after starting, it becomes 137 and continues to deploy endlessly, eventually hitting the DockerHUB limit and not being able to pull images for more than 6 hours.

As a test, I increased the memory and CPU of the API container task, but it did not seem to improve.
# Task STatus
![Image](https://github.com/user-attachments/assets/1bc05bfd-80a7-4d18-be90-57cd7075430a)

@tmokmss Thanks for reply.
please check daemon cloudwatch logs.

```
2025/03/20 01:04:05 pool.go:32: [32m[INFO]init routine pool, size: 10000[0m
2025/03/20 01:04:06 init.go:152: [32m[INFO]dify plugin db initialized[0m
2025/03/20 01:04:06 server.go:31: [31m[PANIC]Failed to create s3 storage: operation error S3: CreateBucket, resolve auth scheme: resolve endpoint: endpoint rule error, Custom endpoint `` was not a valid URI[0m
panic: [PANIC]Failed to create s3 storage: operation error S3: CreateBucket, resolve auth scheme: resolve endpoint: endpoint rule error, Custom endpoint `` was not a valid URI%!(EXTRA *smithy.OperationError=operation error S3: CreateBucket, resolve auth scheme: resolve endpoint: endpoint rule error, Custom endpoint `` was not a valid URI)
goroutine 1 [running]:
github.com/langgenius/dify-plugin-daemon/internal/utils/log.writeLog({0x15fcc6a, 0x5}, {0x1627420?, 0x0?}, 0x1, {0xc0003dfd00, 0x1, 0x1})
	/app/internal/utils/log/log.go:40 +0x326
github.com/langgenius/dify-plugin-daemon/internal/utils/log.Panic(...)
	/app/internal/utils/log/log.go:66
github.com/langgenius/dify-plugin-daemon/internal/server.initOSS(0x2710?)
	/app/internal/server/server.go:31 +0xf2
github.com/langgenius/dify-plugin-daemon/internal/server.(*App).Run(0xc0005e20c0, 0xc0005a0388)
	/app/internal/server/server.go:60 +0xdf
main.main()
	/app/cmd/server/main.go:28 +0xff
```

@tmokmss I tried running it in cldou shell, but got the same result.
```
<html>
<body>
<!--StartFragment-->
 
--
2025/03/20 05:15:33 pool.go:32: [32m[INFO]init routine pool, size: 10000[0m

2025/03/20 05:15:33 [35m/app/internal/db/init.go:17
[0m[31m[error] [0mfailed to initialize database, got error failed to connect to `host=difyonawsstack-postgrescluster53e5bdab-dptr5pn3gi1y.cluster-cs5syq8skg0b.us-east-1.rds.amazonaws.com user=postgres database=dify_plugin`: server error (FATAL: database "dify_plugin" does not exist (SQLSTATE 3D000))
2025/03/20 05:15:35 init.go:152: [32m[INFO]dify plugin db initialized[0m
2025/03/20 05:15:35 server.go:31: [31m[PANIC]Failed to create s3 storage: operation error S3: CreateBucket, resolve auth scheme: resolve endpoint: endpoint rule error, Custom endpoint `` was not a valid URI[0m
panic: [PANIC]Failed to create s3 storage: operation error S3: CreateBucket, resolve auth scheme: resolve endpoint: endpoint rule error, Custom endpoint `` was not a valid URI%!(EXTRA *smithy.OperationError=operation error S3: CreateBucket, resolve auth scheme: resolve endpoint: endpoint rule error, Custom endpoint `` was not a valid URI)
goroutine 1 [running]:
github.com/langgenius/dify-plugin-daemon/internal/utils/log.writeLog({0x15fcc6a, 0x5}, {0x1627420?, 0x0?}, 0x1, {0xc0003dfd00, 0x1, 0x1})
	/app/internal/utils/log/log.go:40 +0x326
github.com/langgenius/dify-plugin-daemon/internal/utils/log.Panic(...)
	/app/internal/utils/log/log.go:66
github.com/langgenius/dify-plugin-daemon/internal/server.initOSS(0x2710?)
	/app/internal/server/server.go:31 +0xf2
github.com/langgenius/dify-plugin-daemon/internal/server.(*App).Run(0xc0001163c0, 0xc000125508)
	/app/internal/server/server.go:60 +0xdf
main.main()
	/app/cmd/server/main.go:28 +0xff
2025/03/20 05:17:59 pool.go:32: [32m[INFO]init routine pool, size: 10000[0m
2025/03/20 05:18:01 init.go:152: [32m[INFO]dify plugin db initialized[0m
2025/03/20 05:18:01 server.go:31: [31m[PANIC]Failed to create s3 storage: operation error S3: CreateBucket, resolve auth scheme: resolve endpoint: endpoint rule error, Custom endpoint `` was not a valid URI[0m
panic: [PANIC]Failed to create s3 storage: operation error S3: CreateBucket, resolve auth scheme: resolve endpoint: endpoint rule error, Custom endpoint `` was not a valid URI%!(EXTRA *smithy.OperationError=operation error S3: CreateBucket, resolve auth scheme: resolve endpoint: endpoint rule error, Custom endpoint `` was not a valid URI)
goroutine 1 [running]:
github.com/langgenius/dify-plugin-daemon/internal/utils/log.writeLog({0x15fcc6a, 0x5}, {0x1627420?, 0x0?}, 0x1, {0xc00045fd00, 0x1, 0x1})
	/app/internal/utils/log/log.go:40 +0x326
github.com/langgenius/dify-plugin-daemon/internal/utils/log.Panic(...)
	/app/internal/utils/log/log.go:66
github.com/langgenius/dify-plugin-daemon/internal/server.initOSS(0x2710?)
	/app/internal/server/server.go:31 +0xf2
github.com/langgenius/dify-plugin-daemon/internal/server.(*App).Run(0xc0000b6120, 0xc000522388)
	/app/internal/server/server.go:60 +0xdf
main.main()
	/app/cmd/server/main.go:28 +0xff
2025/03/20 05:20:20 pool.go:32: [32m[INFO]init routine pool, size: 10000[0m
2025/03/20 05:20:20 init.go:152: [32m[INFO]dify plugin db initialized[0m
2025/03/20 05:20:20 server.go:31: [31m[PANIC]Failed to create s3 storage: operation error S3: CreateBucket, resolve auth scheme: resolve endpoint: endpoint rule error, Custom endpoint `` was not a valid URI[0m
panic: [PANIC]Failed to create s3 storage: operation error S3: CreateBucket, resolve auth scheme: resolve endpoint: endpoint rule error, Custom endpoint `` was not a valid URI%!(EXTRA *smithy.OperationError=operation error S3: CreateBucket, resolve auth scheme: resolve endpoint: endpoint rule error, Custom endpoint `` was not a valid URI)
goroutine 1 [running]:
github.com/langgenius/dify-plugin-daemon/internal/utils/log.writeLog({0x15fcc6a, 0x5}, {0x1627420?, 0x0?}, 0x1, {0xc0005bfd00, 0x1, 0x1})
	/app/internal/utils/log/log.go:40 +0x326
github.com/langgenius/dify-plugin-daemon/internal/utils/log.Panic(...)
	/app/internal/utils/log/log.go:66
github.com/langgenius/dify-plugin-daemon/internal/server.initOSS(0x2710?)
	/app/internal/server/server.go:31 +0xf2
github.com/langgenius/dify-plugin-daemon/internal/server.(*App).Run(0xc0000c04b0, 0xc0000df508)
	/app/internal/server/server.go:60 +0xdf
main.main()
	/app/cmd/server/main.go:28 +0xff
```

--- a/lib/constructs/dify-services/api.ts
+++ b/lib/constructs/dify-services/api.ts
@@ -349,6 +349,8 @@ export class ApiService extends Construct {
         DIFY_INNER_API_URL: `http://localhost:${port}`,
         PLUGIN_WORKING_PATH: '/app/storage/cwd',
         FORCE_VERIFYING_SIGNATURE: 'true',
+        S3_USE_AWS_MANAGED_IAM: 'true',
+        S3_ENDPOINT: `https://s3.${Stack.of(this).region}.amazonaws.com`,
       },
       secrets: {
         DB_USERNAME: ecs.Secret.fromSecretsManager(postgres.secret, 'username'),

